/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/util.py:14: DeprecationWarning: The compiler package is deprecated and removed in Python 3.x.
  from compiler.consts import CO_GENERATOR
E.E.E.E.E.E.E.E.E.E.E.E..........
======================================================================
ERROR: __main__.test_ball_tree(True, 1, 'chebyshev', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1ddd590>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 3, 'chebyshev', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e2b370>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 5, 'chebyshev', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e2e120>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 1, 'euclidean', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e30ae0>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 3, 'euclidean', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e32e80>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 5, 'euclidean', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e35f10>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 1, 'minkowski', {'p': 4})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e384e0>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 3, 'minkowski', {'p': 4})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e3aea0>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 5, 'minkowski', {'p': 4})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e3d520>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 1, 'cityblock', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e3fee0>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 3, 'cityblock', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e42540>

--------------------- >> end captured stdout << ----------------------

======================================================================
ERROR: __main__.test_ball_tree(True, 5, 'cityblock', {})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vanderplas/PythonEnv/default/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "test_binary_tree.py", line 28, in check_neighbors
    dist1, ind1 = bt.query(Y, k, dualtree=dualtree)
  File "binary_tree.pyx", line 289, in binary_tree.BinaryTree.query (binary_tree.c:4389)
    bounds = np.inf + np.zeros(other.node_data.shape[0])
AttributeError: 'binary_tree.BallTree' object has no attribute 'node_data'
-------------------- >> begin captured stdout << ---------------------
<binary_tree.BallTree object at 0x1e44e50>

--------------------- >> end captured stdout << ----------------------

----------------------------------------------------------------------
Ran 33 tests in 0.044s

FAILED (errors=12)
